src/QandaAtom.tsx

```diff
@@ -138,12 +138,13 @@ const buttonRound = css`
     }
 `;
 
+// The important is here to override the default image style which defaults to 100% width
```

Comment:
So I understand this, where is that default style coming from? It should be that specific styles always override default styles from the cascade, we should never need to use `!important`

Reply:
I think it's coming from DCR, I haven't added any `width: 100%` styles anywhere in atoms-rendering